be/tool/beck.c

```diff
@@ -2474,20 +2474,44 @@ static struct cache_slot *cache_insert(struct cache *c,
 	return slot;
 }
 
+static int get_callback(struct m0_btree_cb  *cb,
```

Comment:
please add ctg prefix in newly added functions.

Reply:
done.